stdlib/public/core/Availability.swift

```diff
@@ -24,7 +24,7 @@ public func _stdlib_isOSVersionAtLeast(
   _ minor: Builtin.Word,
   _ patch: Builtin.Word
 ) -> Builtin.Int1 {
-#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+#if (os(macOS) || os(iOS) || os(tvOS) || os(watchOS)) && !SWIFT_RUNTIME_NO_OS_VERSIONING
```

Comment:
These double negatives make my brain go all squishy. Can you change it to be a positive instead (like `SWIFT_RUNTIME_OS_VERSIONING`) to avoid that?

Reply:
Fixed!

Reply:
Great, thank you!